web/modules/custom/custom_services/src/Event.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Drupal\custom_services;
+
+
+use Drupal\node\Entity\Node;
+use Drupal\paragraphs\Entity\Paragraph;
+
+class Event {
+
+  public $event = NULL;
+  public $tickets = [];
+
+  /**
+   * @param $id
+   */
+  public function load($id) {
+    $this->event = Node::load($id);
+    $this->tickets = Paragraph::load($this->event->get('field_tickets')->getValue()[0]['target_id']);
+  }
+
+  /**
+   * @return mixed
+   */
+  public function id() {
+    return $this->event->id();
+  }
+
+  /**
+   * @param $field
+   *
+   * @return mixed
+   */
+  public function __get($field) {
```

Comment:
This function is never used

Reply:
https://github.com/mpndev/shano/blob/extract_services/web/modules/custom/shano_shopping_cart/src/Form/AddTicketToCartForm.php   on line 92. We will use on other pleases too.